client/dfclient/src/gameplay_state.cpp

```diff
@@ -25,6 +25,9 @@ const int FISH_FRAME_HEIGHT = 120;
 const int IMGS_PER_ROW = 10;
 const int IMGS_PER_SPECIES = 80;
 
+const int TILE_WIDTH = 128;
+const int TILE_HEIGHT = 128;
+
```

Comment:
Instead of hardcoding those values, maybe we should instead encode tile with and height in the level file?

Both `<tileset>` and `<tilemap>` tags have `tilewidth` and `tileheight` attributes, so maybe we read some of those in the python script and then encode them into the level file?

@capturcus WDUT?

Reply:
Agreed with @piodul , we should probably encode tile width and height into the level file.